packages/jsii-pacmak/bin/jsii-pacmak.ts

```diff
@@ -153,6 +153,34 @@ import { VERSION_DESC } from '../lib/version';
       default: undefined,
       hidden: true,
     })
+    .option('maven-extra-repository-id', {
+      type: 'string',
+      desc: 'ID of additional repository to add to Maven settings file',
+      defaultDescription: 'No extra repository is registered.',
+      default: undefined,
+      hidden: true,
+    })
+    .option('maven-extra-repository-url', {
+      type: 'string',
+      desc: 'URL of additional repository to add to Maven settings file',
+      defaultDescription: 'No extra repository is registered.',
+      default: undefined,
+      hidden: true,
+    })
+    .option('maven-extra-repository-username', {
+      type: 'string',
+      desc: 'Username for authenticating to extra repository',
+      defaultDescription: 'No extra repository is registered.',
+      default: undefined,
+      hidden: true,
+    })
+    .option('maven-extra-repository-password', {
+      type: 'string',
+      desc: 'Password for authenticating to extra repository',
+      defaultDescription: 'No extra repository is registered.',
+      default: undefined,
+      hidden: true,
+    })
```

Comment:
Passing passwords over CLI arguments is notoriously risky... There's many avenues through which program arguments may be visible by other parties...

I get that these _may_ be "short-lived" credentials here (so less risky to _maybe_ leak somewhere), but I'd still rather have an alternate (safer?) way to provide the password... Not sure what the typical maven practice there is...

This ultimately makes me wonder if we shouldn't instead allow one to provide a "base" XML maven settings file to use, which can include some repos, credentials, etc... This is just an idea, though... I don't know that Maven settings XML documents allow "extending" others without actually copying + modifying them... Which sounds like a lot of unneeded work.